install_config/aggregate_logging.adoc

```diff
@@ -468,81 +468,84 @@ endif::openshift-enterprise[]
 [[scaling-elasticsearch]]
 *Changing the Scale of Elasticsearch*
 
-If you need to scale up the number of Elasticsearch instances your cluster uses
-it is not as simple as changing the number of replicas. This is due to the nature
-of Persistent volumes and how Elasticsearch is configured to store its data and
-recover the cluster.
+If you need to scale up the number of Elasticsearch instances your cluster uses,
+it is not as simple as changing the number of replicas. This is due to the
+nature of persistent volumes and how Elasticsearch is configured to store its
+data and recover the cluster.
 
-Instead, you will need to create a Deployment Configuration for each replica.
+Instead, you will need to create a deployment configuration for each replica.
 
-The Deployer created templates during installation with the Elasticsearch
-configurations provided to it: `logging-es-template` and `logging-es-ops-template`
-if the Deployer was run with `ENABLE_OPS_CLUSTER=true`.
+The deployer created templates during installation with the Elasticsearch
+configurations provided to it: *logging-es-template* and
+*logging-es-ops-template* if the deployer was run with
+`*ENABLE_OPS_CLUSTER=true*`.
```

Comment:
@ahardin-rh @ewolinetz This seems out of place here. I know this is said somewhere else in this topic. Is the para relevant?